### DISCUSSION TITLE
Separate section for national indicators

### DISCUSSION THREAD

**Discussion by LucyGwilliamAdmin** [open-sdg/open-sdg#1400] [Ideas]
While delivering workshop for Kyrgyzstan, they asked if possible to have separate sections for global and national indicators - similar to [Poland's site](http://sdg.gov.pl/)

Since most of the countries who use Open SDG have national indicators, think it might be worth adding functionality for this.

Another way of doing it which I quite like is [Germany's site](https://sustainabledevelopment-germany.github.io/en/), where they have the toggle option in the top-right. 

Maybe Moritz could help get it in to Open SDG?

**Comment by brockfanning** [comment]
@LucyGwilliamAdmin I believe in Germany's case, this is done by having two separate implementations of Open SDG, and linking between them (you'll notice the domain name changes when you go back and forth).

This could potentially be done by supporting a `filter_indicators` (or something) option in the front-matter for goal pages. That could allow the goal page to only display certain indicators, according to some filter. Like:

```
filter_indicators:
    national_or_global: national
```

Of course, we don't recommend using individual files for goal pages any more, since we have the `create_goals` option. So we'd probably want to think about how to we could work that into the `create_goals` option too.

**Comment by brockfanning** [comment]
Reviewed January 2021 - not in next few sprints.
